src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return color;
```

Comment:
write more descriptive implementation. the result should be like "Ball(color=your_color, number=your_number)"